custom_components/adjustable_bed/config_flow.py

```
@@ -179,17 +187,95 @@ async def async_step_bluetooth(
+            detection_result.confidence,
         )
 
         self._discovery_info = discovery_info
         self.context["title_placeholders"] = {"name": discovery_info.name or discovery_info.address}
 
+        # Check if disambiguation is needed (low confidence with alternatives)
+        if detection_result.confidence < 0.7 and detection_result.ambiguous_types:
+            # Build list of all candidate types (detected + alternatives), deduplicated
+            seen: set[str] = set()
+            disambiguation_types: list[str] = []
+            for t in [bed_type] + list(detection_result.ambiguous_types):
```

Ruff (0.14.14)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)